test/unit/windowHandlers.test.js

```diff
@@ -251,17 +251,6 @@ describe("Window handlers test", function () {
                 ],
             });
 
-            try {
```

Comment:
Had to remove this has this was failing. I checked the code and `createPasswordlessCode` no longer saves the response to localStorage unless the response.status is `OK`.